### ISSUE TITLE
Can't build javafx-maven-plugin with JDK8

### ISSUE THREAD

**Issue by dwhitla** [openjfx/javafx-maven-plugin#86]
The target and source language levels specified in the POM are both Java8, which is consistent with expectations for a maven plugin, but the plugin cannot be built with any JDK prior to 12 due to test dependencies with class file version 55.0. If you want to test using another JDK you should fork a new process from the main build reactor and run it under the alternate JDK.

**Comment by dwhitla** [comment]
FWIW I think you should be testing the execution of _the plugin_ here, not a JavaFX project built using the plugin.

**Comment by rruffer** [comment]
For java 8 use this: https://github.com/javafx-maven-plugin/javafx-maven-plugin

**Comment by dwhitla** [comment]
The issue is not building Java FX projects on Java 8. It is building the Java FX Maven Plugin on Java 8. All maven plugins are supposed to be buildable on Java 8.

**Comment by abhinayagarwal** [comment]
This is a valid issue and should be resolved.

@dwhitla In PR #50, you have [commented](https://github.com/openjfx/javafx-maven-plugin/issues/50#issuecomment-630446353) that you are already working on a fix. If yes, can you create a WIP PR?

**Comment by dwhitla** [comment]
> @dwhitla In PR #50, you have [commented](https://github.com/openjfx/javafx-maven-plugin/issues/50#issuecomment-630446353) that you are already working on a fix. If yes, can you create a WIP PR?

Yes. I actually had it all fixed and then decided it was poor form to fix two issues in a single PR. So I am in the process of separating the changes as two separate fixes - one for issue #50 and one for this issue. This would have been completed 3 weeks ago but with the current economic chaos I have to focus on paying work at the expense of unpaid contributions.

**Comment by abhinayagarwal** [comment]
Thanks for the update. If you would like to open a WIP PR, I could help review and contribute to the same PR.

**Comment by dwhitla** [comment]
I will have a look at where I left it now and see how quickly I can get something in.

**Comment by dwhitla** [comment]
For information, the fix for this issue is to use the maven-invoker-plugin rather than the test harness for testing.
